src/lib/constants.js

```diff
@@ -1,10 +1,10 @@
-export const CHANNEL_NAME = "Themer";
+export const CHANNEL_NAME = "FirefoxColor";
 
 export const DOWNLOAD_FIREFOX_URL =
-  "https://www.mozilla.org/firefox/new/?utm_campaign=themer-acquisition&utm_medium=referral&utm_source=themer.firefox.com";
+  "https://www.mozilla.org/firefox/new/?utm_campaign=firefoxcolor-acquisition&utm_medium=referral&utm_source=color.firefox.com";
```

Comment:
I just went ahead and changed these parameters for GA - is that copacetic? /cc @johngruen

Reply:
`utm_campaign` certainly is, so long as we haven't production shipped yet.

`utm_source` should probably come from an environment variable?